tests/factories.py

```diff
@@ -718,3 +719,69 @@ def _create(cls, *args, **kwargs):
             )
         )
         return super(NodeLicenseRecordFactory, cls)._create(*args, **kwargs)
+
+
+def render_generations_from_parent(parent, creator, num_generations):
+    current_gen = parent
+    for generation in xrange(0, num_generations):
+        next_gen = NodeFactory(
+            parent=current_gen,
+            creator=creator,
+            title=fake.science_sentence(),
+            description=fake.science_paragraph()
+        )
+        current_gen = next_gen
+    return current_gen
+
+
+def render_generations_from_node_structure_list(parent, creator, node_structure_list):
+    new_parent = None
+    for node_number in node_structure_list:
+        if isinstance(node_number, list):
+            render_generations_from_node_structure_list(new_parent or parent, creator, node_number)
+        else:
+            new_parent = render_generations_from_parent(parent, creator, node_number)
+    return new_parent
+
+
+def create_fake_user():
+    email = fake.email()
+    name = fake.name()
+    parsed = impute_names(name)
+    user = UserFactory(username=email, fullname=name,
```

Comment:
Do we need all this boilerplate? Why can't we just use `UserFactory()` to create the instance directly? (any arguments not provided should be taken from the defaults)